src/Controls/src/Core/Editor/Editor.cs

```diff
@@ -66,40 +64,6 @@ public EditorAutoSizeOption AutoSize
 			set { SetValue(AutoSizeProperty, value); }
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Editor.xml" path="//Member[@MemberName='FontAttributes']/Docs/*" />
-		public FontAttributes FontAttributes
```

Comment:
Do we also need a new for this? It will blow up for net7 libraries. everything needs a new!

Reply:
I m not sure i understand why , this property will still exist no? if we look at Placeholder for example is similar implementation.

Reply:
I suppose with more testing we will see. We can always add back in RC 2.